utils/sound_upload.py

```diff
@@ -66,10 +67,31 @@ def _remove_user_uploads_folder_if_empty(user):
     remove_empty_user_directory_from_mirror_locations(user_uploads_dir)
 
 
-def create_sound(user, sound_fields, apiv2_client=None, process=True, remove_exists=False):
+def create_sound(user,
```

Comment:
changed format here for some reason?

Reply:
Because Ia added an extra item and it looked cleaner this way.